.github/actions/setup-node/action.yml

```diff
@@ -0,0 +1,47 @@
+name: "Node setup"
```

Comment:
Yeah, I suppose it could be either. In this context, I had considered "node setup" to be configuring the host on which this action is running. I can change it if folks think a different term would be more clear.